sql/slow-query.md

```diff
@@ -8,89 +8,210 @@ category: user guide
++-------------+-----------------------------+------------------------------------------------------------------+
+1 row in set
+Time: 0.007s
+tidb > select query, query_time from INFORMATION_SCHEMA.`SLOW_QUERY` where digest="4751cb6008fda383e22dacb601fde85425dc8f8cf669338d55d944bafb46a6fa";
++-----------------------------+-------------+
+| query                       | query_time  |
++-----------------------------+-------------+
+| select * from t1 where a=1; | 0.302558006 |
+| select * from t1 where a=2; | 0.401313532 |
++-----------------------------+-------------+
+2 rows in set
+```
 
-The IDs of the tables involved in the statement.
+## Query the slow query whose stats is pseudo
 
-#### `index_ids`
+```sql
+tidb > select query, query_time, stats from INFORMATION_SCHEMA.`SLOW_QUERY` where is_internal=false and stats like('%pseudo%');
++-----------------------------+-------------+---------------------------------+
+| query                       | query_time  | stats                           |
++-----------------------------+-------------+---------------------------------+
+| select * from t1 where a=1; | 0.302558006 | t1:pseudo                       |
+| select * from t1 where a=2; | 0.401313532 | t1:pseudo                       |
+| select * from t1 where a>2; | 0.602011247 | t1:pseudo                       |
+| select * from t1 where a>3; | 0.50077719  | t1:pseudo                       |
+| select * from t1 join t2;   | 0.931260518 | t1:407872303825682445,t2:pseudo |
++-----------------------------+-------------+---------------------------------+
+```
 
-The IDs of the indexes involved in the statement.
+#### Parse other TiDB slow log files
 
-#### `sql`
+Currently, when you query `INFORMATION_SCHEMA.SLOW_QUERY` table, only this file will be parsed. This file corresponds to the slow query name set by `slow-query-file` in the configuration file and refers to "tidb-slow.log" by default. If you want to parse other log files, you can set the session variable `tidb_slow_query_file` to the specific file path, and then query INFORMATION_SCHEMA.SLOW_QUERY` table to parse the slow log file according to the set path.
+```sql
+/* Set the slow log file path to facilitate so that other slow log files will be easy to be parsed. The scope of the tidb_slow_query_file variable is session. */
+tidb > set tidb_slow_query_file="/path-to-log/tidb-slow.log"
+Query OK, 0 rows affected
+Time: 0.001s
+```
 
-The SQL statement.
+Currently, `INFORMATION_SCHEMA.SLOW_QUERY` only supports parsing one single slow log file. If the slow log file which exceeds a certain size is logrotated into multiple files, querying `INFORMATION_SCHEMA.SLOW_QUERY` will only parse one file. We will improve this later.
+
+### Parse TiDB slow log with pt-query-digest
+
+You can analyze TiDB slow log with pt-query-digest. pt-query-digest 3.0.13 version or later is recommended. The examples are as followed:
```

Comment:
```suggestion
TiDB slow logs can be analyzed by `pt-query-digest`, for example:
```

Reply:
Why do you add "pt-query-digest 3.0.13 version or later is recommended"? @huangxiuyan @IzabelWang

Reply:
"as followed" is wrong. We usually say "as follows". @huangxiuyan @IzabelWang

Reply:
@CaitinChen , The Chinese version has recommend "pt-query-digest 3.0.13 version or later".  Because sometimes lower version of `pt-query-digest` may have some problem.

Reply:
@crazycs520 Yes. I have updated it.